assets/src/components/swingsView.tsx

```diff
@@ -105,19 +105,27 @@ const SwingsTable = ({
         <thead className="m-swings-view__table-header">
           <tr>
             <th className="m-swings-view__table-header-cell">
-              Swing On <span className="swing-on-time">Time</span>
+              <div>
+                Swing On
+                <div className="subheaders">
```

Comment:
@lemald let's stick to BEM conventions for class names as we mostly have throughout Skate: https://css-tricks.com/bem-101/. it's wordier but is a big help in keeping CSS organized.

Reply:
We do actually break this convention a few places in Skate (for instance the classes `early` and `late` on vehicle icons)... that said, just because we do it one place doesn't necessarily mean it's best practice, so I'll go ahead and change this tomorrow. 🙂

Reply:
It's a helpful practice, a workaround for the fact that all class names in CSS go into a global namespace effectively, so it's very easy to accidentally apply CSS rules to elements due to a collision in class names, leading to all kinds of layout messes big and small.

Reply:
Hmm, that page you linked to discusses an interesting pattern for using SASS with BEM that we don't seem to take advantage of. I wonder if that's worth exploring in the future.

Reply:
I also realized that I hadn't adequately cleaned up some of the old mobile-specific CSS that's now broken, and in fact it looks like I can remove most of it.